packages/playground/COMPONENT_SPECS.md

```
@@ -0,0 +1,333 @@
+
+---
+
+## Water / Puddle Interaction
+
+### Visual Requirements
+1. Water surface is REFLECTIVE
+2. Water surface shows ripple distortion
+3. Character interaction creates splash
+4. Shallow water (puddle) vs deep water visually different
+
+### Acceptance Checklist
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)